virt/virtual_machines/advanced_vm_management/virt-configuring-a-watchdog.adoc

```diff
@@ -0,0 +1,15 @@
+[id="virt-configuring-a-watchdog"]
+= Configuring a watchdog
+include::modules/virt-document-attributes.adoc[]
+:context: virt-configuring-a-watchdog
+
+toc::[]
+
+Expose a watchdog device on a virtual machine instance (VMI), alongside liveness and readiness probes, to ensure the operating system is running properly.
+
+== Prerequisites
+
+* Kernel support for `i6300esb` inside the guest
```

Comment:
Can you provide some information? This would be a good place to look for info to link to.